tests/utils/datatypes/date_datatype_tests.py

```diff
@@ -18,19 +20,20 @@ class DateDataTypeTests(ArchesTestCase):
     ]
 
     def test_string_validate(self):
-        datatype = DataTypeFactory().get_instance("date")
         for date in self.sample_dates:
             with self.subTest(input=date):
-                errors = datatype.validate(date)
+                errors = self.datatype.validate(date)
                 self.assertEqual(len(errors), 0)
 
     def test_tile_transform(self):
-        datatype = DataTypeFactory().get_instance("date")
-        self.sample_dates.append(
-            datetime.strptime("2025-01-01 00:00:00", "%Y-%m-%d %H:%M:%S")
-        )
-        for date in self.sample_dates:
+        python_dates = [
+            datetime.datetime.strptime("2025-01-01 00:00:00", "%Y-%m-%d %H:%M:%S"),
+            datetime.date(2025, 1, 1),
+        ]
+        for date in (*self.sample_dates, *python_dates):
```

Comment:
These dates were persisting on the class array after this test, so I segregated them out.